x-pack/plugins/index_management/public/store/reducers/table_state.js

```diff
@@ -11,9 +11,10 @@ import {
   pageSizeChanged,
   sortChanged,
   showSystemIndicesChanged,
+  toggleChanged,
 } from '../actions';
 
-const defaultState = {
+export const defaultTableState = {
```

Comment:
Should we add `toggles: {}` to `defaultTableState`?

Reply:
don't need to as the state is initialized with it set.